docs/integrations/third-party-libraries.md

```diff
@@ -23,6 +23,7 @@ This can be done using third-party libraries created by the community.
 - [leiten-zustand](https://github.com/hecmatyar/leiten-zustand) — Cleans your store from boilerplate for requests and data transformation.
 - [mobz](https://github.com/2A5F/Mobz) — Zustand-style MobX API.
 - [ngx-zustand](https://github.com/JoaoPauloLousada/ngx-zustand) - A Zustand adapter for Angular.
+- [persist-and-sync](https://github.com/mayank1513/persist-and-sync) - Zustand middleware to easily persist and sync Zustand state between tabs / windows / iframes (SameOrigin)
```

Comment:
You can add a full stop at the end and rephrase "SameOrigin" to something like "for apps from the same origin."